pages/linux/httpie.md

```diff
@@ -0,0 +1,31 @@
+# httpie
+
+> A human-friendly CLI HTTP tool.
```

Comment:
We usually avoid using subjective definitions. How about `A user friendly command line HTTP tool` ?

Reply:
OK. I got it.